app-admin/graylog2/files/initd-r2

```diff
@@ -0,0 +1,31 @@
+#!/sbin/openrc-run
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+command="/usr/bin/java"
+command_args="${JAVA_OPTS} -jar ${GRAYLOG_INSTALL_DIR}/graylog.jar server -f ${GRAYLOG_CONFIG_FILE} --no-pid-file ${GRAYLOG_OPTIONS}"
+command_background="true"
+command_user="${GRAYLOG_USER}:${GRAYLOG_GROUP}"
+pidfile="/run/${RC_SVCNAME}.pid"
+
+output_log="${GRAYLOG_LOG_DIR}/graylog.stdout.log"
+error_log="${GRAYLOG_LOG_DIR}/graylog.stderr.log"
+directory="${GRAYLOG_INSTALL_DIR}"
+required_files="${GRAYLOG_CONFIG_FILE}"
+
+depend() {
+	need net
```

Comment:
Do we really `need net`? See https://bugs.gentoo.org/439092

Reply:
Graylog is a central logging system, it also connects to elasticsearch/mongodb, so it really is network dependant and fails to work otherwise.

Reply:
See the bug's first comment, `need net` is not doing what people expect. Any interface, including lo, satisfy `need net`. You will probably want to change into `use dns` for the part connecting to es/mongodb and tell people they have to add their specific net service to /etc/conf.d/graylog if they bind service to a specific interface/IP.